cypress/support/pages/home.pageObject.js

```diff
@@ -6,6 +6,10 @@ class homePageObject extends PageObject {
   get usernameLink() {
     return cy.getByDataCy('/profile/');
```

Comment:
Use the correct attribute name for data-cy, replace '/profile/' with 'profile-link' or an appropriate descriptive name.